docs/more-pushmsg.md

```diff
@@ -83,3 +83,36 @@ switch ($code) {
         echo "<strong>Hmm Something Went Wrong or HTTP Status Code is Missing</strong>";
 }
 ```
+
+### Node.js - Using Axios
+
+```javascript
+////////////////////////////////////////
+// Send Push Message to Gotify Server //
+////////////////////////////////////////
+
+var url = "http://localhost:8008/message?token=<apptoken>";
+var bodyFormData = {
+  title: "Hello from Javascript",
+  message: "Test Push Service from Node.js",
+  priority: 5
+};
+axios({
+  method: "post",
+  headers: {
+    "Content-Type": "application/json"
+  },
+  url: url,
+  data: bodyFormData
+})
+  .then(function(response) {
+    console.log(response.data);
+  })
+  .catch(function(error) {
+    if (!error.response) {
+      console.log("Gotify API URL is Missing");
```

Comment:
Is this really true? I think the response is `undefined` if the url that was defined isn't reachable or does not exist.

Reply:
if URL was not in Correct format 
if not reachable the URL

Reply:
Could you `console.log(error)` instead? This should result in a sufficient error message.

Reply:
sure let me change